modules/EventBrowser/source/inc/falaise/snemo/view/status_bar.h

```diff
@@ -65,7 +73,7 @@ namespace snemo {
         status_bar();
 
         /// Destructor
-        ~status_bar();
+        virtual ~status_bar();
```

Comment:
Put cons/destructors as the first member functions after the initial `public:` statement to follow the common declaration order.